frontend/src/metabase/visualizations/components/LegendItem.styled.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+import { space } from "metabase/styled-components/theme";
+
+export const IconContainer = styled.div`
+  padding-right: ${space(0)};
```

Comment:
<3

Reply:
These helpers are very handy, thank you @gusaiani